src/editmap.cpp

```diff
@@ -1118,18 +1117,18 @@ int editmap::edit_trp()
         for ( int t = tshift; t <= tshift + tmax; t++ ) {
             mvwprintz(w_picktrap, t + 1 - tshift, 1, c_white, "%s", padding.c_str());
             if ( t < num_trap_types ) {
+                auto &tr = trap_id( t ).obj();
```

Comment:
This part doesn't seem to work correctly. It adds `tr_null`, but cuts off the last trap on the list, so `trap_id( 1 )` probably is the id of `tr_null`. Might cause problems in some rare cases (legacy saves?).